infra.bs

```diff
@@ -516,17 +516,20 @@ contains, in the range 0x61 (a) to 0x7A (z), inclusive, by 0x20.
  </li>
 </ol>
 
+<p>"<var>a</var> <dfn export for="byte sequence">starts with</dfn> <var>b</var> can be used as a
+synonym for "<var>b</var> is a <a for="byte sequence">prefix</a> of <var>a</var>".
```

Comment:
Let's use `<dfn lt>` as we do elsewhere?

Reply:
You mean, on the prefix definition? I think in this case we should have separate dfns, since they are literally the opposite of each other...

Reply:
Oh wow, I missed that. Agreed.

Reply:
What do you think about not using a and b but instead _prefix_ and _input_ or some such?

Reply:
That's a great idea, will change.